versions.properties

```diff
@@ -1,5 +1,5 @@
 # Scala version used for bootstrapping (see README.md)
-starr.version=2.13.0-M5
+starr.version=2.13.0-pre-6e0cba7-SNAPSHOT
```

Comment:
hmm. should this be the non-SNAPSHOT version? it looks like we don't have sbt set up to point to `scala-integration` on artifactory.

Reply:
STARR must be on Maven Central. someone on the Scala team has to run the necessary publishing job on Travis

Reply:
Where to find this version?

Reply:
oh! I assumed that when it was done in the past, it was done through `scala-integration`. and my apologies, looking at the ticket; I didn't realize that it worked after a bootstrap.

Reply:
> Where to find this version?

PR validation snapshots (for as-yet-unmerged PRs) are at https://scala-ci.typesafe.com/artifactory/scala-pr-validation-snapshots/ , "nightlies" for merged PRs (not really "nightly", one is built for every merged PR) https://scala-ci.typesafe.com/artifactory/scala-integration/